floor/floor/processor/constants.py

```diff
@@ -5,7 +5,7 @@
 
 # Maximum value for any component color in a pixel. As currently defined,
 # colors have a 10-bit range of `[0, 1024]`
-COLOR_MAXIMUM = 1024
+COLOR_MAXIMUM = 1023
```

Comment:
This threw me for a while.  Full brightness patterns were clipping to zero on output and producing a blank floor.

Reply:
yikes! good catch. can you update the comment above as well.

Reply:
Updated